sdk/src/server-api/sc/networking/clients/LobbyClient.java

```diff
@@ -267,23 +267,21 @@ protected void request(ProtocolMessage request, Class<? extends ProtocolMessage>
     send(request);
   }
 
-  @SuppressWarnings("unchecked")
   protected RequestResult blockingRequest(ProtocolMessage request,
                                           Class<? extends ProtocolMessage> response) throws InterruptedException {
-    final RequestResult requestResult = new RequestResult();
+    final RequestResult[] requestResult = {null};
```

Comment:
Why do you need an array here?

Reply:
Don't ask... parallel code...

Reply:
This code is over 10 years old, better stay with what IDEA suggested and change nothing else :sweat_smile:

Reply:
A comment explaining that would be nice, so that we know that it should be refactored in the future.